tests/explorer.api.tests/.vcr/ExploreTests.FailWithDeleteMethod.yaml

```diff
@@ -0,0 +1,20 @@
+httpInteractions:
+- request:
+    method: DELETE
+    uri: http://localhost/explore
+    body:
+      encoding: UTF8-8BIT
+      text: |-
+        {"ApiKey":"SFMyNTY.g3QAAAACZAAEZGF0YW0AAAAkNWEwOGU5MGUtODJjOC00ZmY2LTkyZmItNGJmN2E2ZTY3YjdhZAAGc2lnbmVkbgYA-oOX9W8B.55CiWZ7Jx1oUBq6t-rHU_VGA1jPtIMzHGzgq49Xcrm8","TableName":"loans","ColumnName":"amount","DataSourceName":"gda_banking"}
```

Comment:
This should be removed from all yaml files. (And remember to cancel this and generate a fresh api key!)

Reply:
I think that there are two possible solutions to fix this.
1. We could cache the VCR cassette files in Travis CI, and then we will not have to add them in git, so we can keep the API keys in the VCR cassettes. I will have to investigate how to configure the Travis CI cache correctly.
2. We could move the API key into the header of the request (just like we have the `auth-token` for Aircloak API), and then filter it out when serializing the `CacheResult` to the VCR cassette file, just as you suggested above.

Reply:
This was solved by caching the VCR cassettes in Travis CI.